guidelines/supportability.md

```diff
@@ -1,4 +1,46 @@
-# What does it mean to be Tech Preview
+# Supportability Commitments
+
+This document aims to cover the support/compatibility expectations associated with features and apis.  It also covers how to
+manage those expectations with respect to upstream features/apis that may be at an alpha or beta level.
+
+At a high level, there are only two official classifications for APIs in OpenShift:
+
+* Tech Preview - The bulk of this document address this category
+* Generally Available(GA) - Our commitments to GA features and APIs are covered in our official [product](https://docs.openshift.com/container-platform/4.13/rest_api/understanding-compatibility-guidelines.html) [documentation](https://docs.openshift.com/container-platform/4.13/rest_api/understanding-api-support-tiers.html)
+
+Within the GA category, there are tiers of API support which describe when an API is allowed to change or be removed, which you can read about from the links
+above.
+
+Upstream [defines](https://kubernetes.io/blog/2020/08/21/moving-forward-from-beta/) 3 tiers of maturity:
+* alpha - off by default, can change freely, no commitment to help migrate a user to new versions
+* beta - on by default, still may change freely, but higher expectation that a GA version will arrive and users *may* be aided in a migration to a GA version
+* GA/Stable  
+
+When bringing an alpha or beta api/feature downstream, we need to carefully consider the implications and support expectations.
+
+## Enabling Upstream Alpha features
+
+There are significant risks associated with exposing customers to Alpha-level upstream features:
+* The api/feature may change in a way that breaks customer expectations/scripts/workloads
+* The api/feature may change in a way that we cannot migrate customer workloads/configuration/resources
+* The api/feature may be removed entirely in a future k8s version leaving us having to carry patches or manually help customers to move forward which may even require an entirely new cluster installation
+
+Given these risks, alpha features **MUST NOT** be turned on downstream unless behind a feature gate that can only be enabled via the TechPreviewNoUpgrade 
+or CustomNoUpgrade featuresets.  If you/your team intend to enable(without a TechPreview gate) an upstream alpha feature, downstream, you must bring the request to 
+the staff engineering group for discussion.  This can be done by adding it to the agenda of an appropriate themed architecture calls(this may not get you approval 
```

Comment:
```suggestion
the staff engineering group for discussion.  This can be done by adding it to the agenda of an appropriate themed architecture calls (this may not get you approval 
```